cmd/flags_node.go

```diff
@@ -68,9 +69,12 @@ func ParseNodeFlags(ctx context.Context, cmd *cobra.Command, network p2p.Network
 			return ctx, err
 		}
 		cfg, err := nodebuilder.LoadConfig(filepath.Join(expanded, "config.toml"))
-		if err == nil {
-			ctx = WithNodeConfig(ctx, cfg)
+		if err != nil {
+			if !errors.Is(err, os.ErrNotExist) {
+				return ctx, err
+			}
 		}
+		ctx = WithNodeConfig(ctx, cfg)
```

Comment:
this will panic if os.ErrNotExist

Let's instead do this: 

if os.ErrNotExist, we should `cfg = nodebuilder.DefaultConfig()` and also `log.Warn` that no config was found in given path(log out the path), initialising with default config

Reply:
Yup makes sense